analyzers/src/SonarAnalyzer.Common/Rules/BypassingAccessibilityBase.cs

```diff
@@ -18,24 +18,26 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using SonarAnalyzer.Common;
 using SonarAnalyzer.Helpers;
 
 namespace SonarAnalyzer.Rules
 {
-    public abstract class BypassingAccessibilityBase<TSyntaxKind> : SonarDiagnosticAnalyzer
+    public abstract class BypassingAccessibilityBase<TSyntaxKind> : TrackerHotspotDiagnosticAnalyzer<TSyntaxKind>
```

Comment:
This is not a Hotspot, it is a code smell.

Reply:
But it uses a lot of functionality from the base class so it's handy to use it as well. As in the other comment, I plan to rename those base classes in the future. This will be probably something like `TrackerDiagnosticAnalyzer` and the `HotspotDiagnosticAnalyzer` will be something like `ConfigurableDiagnosticAnalyzer`